Makefile

```diff
@@ -12,6 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+docker/generated.mk: docker/generate_makefile.py $(wildcard fuzzers/*/variants.yaml)
+	/usr/bin/env python3 $< > $@
```

Comment:
If you'd like to make sure the yaml module to be installed when this runs, you can make this rule depend on `install-dependencies`, and do `source ${VENV_ACTIVATE} && python3 $< > $@` instead.